docs/getting-started/complete.mdx

```diff
@@ -0,0 +1,247 @@
+# Complete Example
+
+Let's add two more components, `CustomCard` and `CustomTextField`, with different use-cases for both the manual and generator approaches.
```

Comment:
I would call this Page "Complete Example". Right now it's called "Complete" in the sidebar.

Reply:
Done